src/parser/mod.rs

```diff
@@ -10290,7 +10290,7 @@ impl<'a> Parser<'a> {
         Ok(ExprWithAlias { expr, alias })
     }
 
-    fn parse_expr_with_alias(&mut self) -> Result<ExprWithAlias, ParserError> {
+    pub fn parse_expr_with_alias(&mut self) -> Result<ExprWithAlias, ParserError> {
```

Comment:
Could you please also add some documentation to this function (perhaps with an example or two) now that it will become part of the public API?

Reply:
Sorry for missing that piece of document, I have updated it.